src/include/catalog/pg_proc.dat

```diff
@@ -8515,9 +8515,9 @@
 { oid => '2508', descr => 'constraint description with pretty-print option',
   proname => 'pg_get_constraintdef', provolatile => 's', prorettype => 'text',
   proargtypes => 'oid bool', prosrc => 'pg_get_constraintdef_ext' },
-{ oid => '8024', descr => 'get CREATE statement for DOMAIN',
+{ oid => '8024', descr => 'get CREATE statement for DOMAIN with pretty option',
   proname => 'pg_get_domain_ddl', prorettype => 'text',
-  proargtypes => 'regtype', prosrc => 'pg_get_domain_ddl' },
+  proargtypes => 'regtype bool', prosrc => 'pg_get_domain_ddl_ext' },
```

Comment:
```suggestion
  proargtypes => 'regtype bool', prosrc => 'pg_get_domain_ddl_ext' },
{ oid => '8025', descr => 'get CREATE statement for DOMAIN',
  proname => 'pg_get_domain_ddl', prorettype => 'text',
  proargtypes => 'regtype', prosrc => 'pg_get_domain_ddl_ext' },
```

Reply:
this instead of adding the create function, better to have all teh changes in one place, maybe...